model/task/task.go

```diff
@@ -2344,6 +2371,12 @@ func (t *Task) GetDisplayStatus() string {
 	return t.DisplayStatus
 }
 
+// FindDisplayStatus publicly exports findDisplayStatus
+// TODO: replace all instances of findDisplayStatus, probably
```

Comment:
Is this TODO meant to be done in this PR or in a follow-up?

Reply:
I think a followup is appropriate. If you think this makes sense I will file a ticket!

Reply:
Either afollow-up PR or a separate ticket SGTM!